pkg/operator/k8sutil/deployment.go

```diff
@@ -79,11 +79,11 @@ func UpdateDeploymentAndWait(context *clusterd.Context, deployment *apps.Deploym
 	}
 
 	// wait for the deployment to be restarted
-	sleepTime := 2
-	attempts := 30
+	sleepTime := 1
+	attempts := 60
 	if original.Spec.ProgressDeadlineSeconds != nil {
 		// make the attempts double the progress deadline since the pod is both stopping and starting
-		attempts = 2 * (int(*original.Spec.ProgressDeadlineSeconds) / sleepTime)
+		attempts = 2 * (int(*original.Spec.ProgressDeadlineSeconds) / 2)
```

Comment:
Do we intend to effectively halve the total retry loop time in the case that ProgressDeadlineSeconds != nil? The semantics of the change seem to revolve around shortening the retry period, not the entire loop duration.